.github/workflows/build.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python }}
-          cache: 'pip'
-          cache-dependency-path: setup.py
+        #   cache: 'pip'
+        #   cache-dependency-path: setup.py
       - name: Install packages
         run: pip install torch && pip install -e .[dev]
```

Comment:
todo: need to revisit why this is failing

https://github.com/arangoml/pyg-adapter/actions/runs/6422719501/job/17439790862

Reply:
Update: https://github.com/arangoml/pyg-adapter/pull/23/commits/4925a49edecfb3241cad7b78c5549455e6fe7fcb seemed to solve the issue

Reply:
Cleaned up in [92cd0c0](https://github.com/arangoml/pyg-adapter/pull/23/commits/92cd0c0076ba8dcdc97aabf8d42b01904d486348)